lib/fog/rackspace/requests/compute_v2/create_server.rb

```diff
@@ -96,12 +97,16 @@ def create_server(name, image_id, flavor_id, min_count, max_count, options = {})
                 Fog::Logger.warning(":boot_volume_id overrides :boot_image_id!")
               end
 
+              if options[:boot_volume_size]
```

Comment:
Good question. @mdarby what do you think? I can see this being useful, but I don't want to end up with the command being too noisy. I guess at logger level you could filter these easily enough though.